libipl/p10/ipl0.C

```diff
@@ -589,11 +595,18 @@ static int ipl_sbe_start(void)
 
 				fapirc = p10_start_cbs(proc, true);
 				if (fapirc == fapi2::FAPI2_RC_SUCCESS) {
+					// Update Primary processor SBE state to check cfam.
+					// Boot error callback is only required for failure.
+					ipl_set_sbe_state(proc, SBE_STATE_CHECK_CFAM);
+
 					if (!ipl_sbe_booted(proc, 25)) {
 						ipl_log(IPL_ERROR, "SBE did not boot\n");
 						err_type = IPL_ERR_SBE_BOOT;
 					}
 					else {
+						// Update Primary processor SBE state to booted
+						// Boot error callback is only required for failure.
+						ipl_set_sbe_state(proc, SBE_STATE_BOOTED);
```

Comment:
is break at line 616 correct? do we need to break assuming if err_type is IPL_ERR_OK?

Reply:
yes , in no cronus moide , this procedure execution required only for master processor., need to skip from pdbg target loop